connectivity/check/deployment.go

```diff
@@ -451,9 +455,83 @@ func (ct *ConnectivityTest) validateDeployment(ctx context.Context) error {
 		}
 	}
 
+	for _, cp := range ct.ciliumPods {
+		err := ct.waitForIPCache(ctx, cp)
+		if err != nil {
+			return fmt.Errorf("Pod identities not available in Cilium ipcaches: %s", err)
```

Comment:
Might be worth to log the Cilium pod where entry wasn't found for debugging purposes.

Reply:
`err` already contains the Cilium pod name. I'll make this more explicit by simply returning `err` here.